src/qmplay2/headers/Playlist.hpp

```diff
@@ -36,6 +36,15 @@ class Playlist
 			queue(0), GID(0), parent(0)
 		{}
 
+		void clear()
```

Comment:
This can go to `cpp`, but anyway, isn't better to use `entry = Entry()` to reset variables :) ?